frontend/admin/src/js/stories/Pool.stories.tsx

```diff
@@ -7,11 +7,11 @@ import {
   fakeUsers,
   fakePools,
 } from "@common/fakeData";
+import { OperationalRequirementV2 } from "@common/constants/localizedConstants";
```

Comment:
```suggestion
import { OperationalRequirementV1 } from "@common/constants/localizedConstants";
```

ah this should be V1 to match the Pool components

Reply:
I think leaving v2 in the Pool forms is fine - we're not expecting any new pools to be created with v1.

Reply:
K, will leave, and ensure update story matches the choice. thanks!